string running_status
string user_id
string task_id
int32 trial
string[] task_ids
